tools/docker/Dockerfile

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2019 by Cornel Izbasa. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM ubuntu:bionic
+ARG REVISION
+ARG B_ICEORYX_BUILD
+ENV ICEORYX_BUILD=$B_ICEORYX_BUILD
+
+RUN apt update && apt install -y \
+        cmake \
+        libacl1-dev \
+        libncurses5-dev \
+        pkg-config \
+        screen \
+        git
+
+ADD . /
+
+WORKDIR /iceoryx
+
+RUN ./tools/iceoryx_build_test.sh \
```

Comment:
In your original version (https://github.com/Mr-Slippery/iceoryx-dockerfiles/blob/master/Dockerfile) you were cloning iceoryx before executing this script. How is the script coming in the docker container?

Reply:
Hi @mossmaurice, indeed, initially I was cloning, but for this "integrated version", I thought it makes sense to build and include into the container the current source that is cloned. This makes it into the container via `ADD . /` in the Dockerfile.

Reply:
But now that you mention it, there is perhaps an issue here, maybe some unintended things make it inside as well, I'll check and fix this.

Reply:
Updated now, indeed, there was an issue, I was going to the parent of top-level `iceoryx` and adding everything into Docker `/`. Now I just go to `iceoryx` and add its content into Docker `/iceoryx`. Rebased branch, melded changes into existing commit. Thanks for the catch!